index.js

```diff
@@ -0,0 +1,51 @@
+let books = JSON.parse(localStorage.getItem('books'));
```

Comment:
[REQUIRED] You should have a collection that stores books. Then you add that collection to the local storage. For example:
`const books = [];`